net/strongswan/files/swanctl.init

```diff
@@ -210,38 +296,40 @@ parse_ike_proposal() {
 	export -n "$var=$crypto"
 }
 
-config_conn() {
-	# Generic ipsec conn section shared by tunnel and transport
+config_child() {
+	# Generic swanctl.conf children section shared by tunnel and transport
 	local config_name="$1"
 	local mode="$2"
 
-	local local_subnet
-	local local_nat
+	local priority
+	local ah_proposal
+	local esp_proposal
 	local updown
 	local firewall
-	local remote_subnet
-	local lifetime
-	local dpdaction
+	local life_time
```

Comment:
I don't want to change the parameter names and break existing configurations.  If that were to happen at all, it would be via its own commit.